packages/components/src/Payments/PaymentsEth.tsx

```diff
@@ -0,0 +1,177 @@
+import * as React from "react";
+import { Mutation, MutationFunc } from "react-apollo";
+import { EthAddress } from "@joincivil/core";
+import { PAYMENTS_ETH_MUTATION } from "./queries";
+import { UsdEthConverter } from "../";
+import {
+  PaymentDirectionsStyled,
+  PaymentEthLearnMore,
+  PaymentBtn,
+  PaymentAmountEth,
+  PaymentTypeLabel,
+} from "./PaymentsStyledComponents";
+import {
+  PayWithEthText,
+  WhyEthInfoText,
+  WhatIsEthInfoText,
+  CanUseCVLInfoText,
+  ConnectWalletWarningText,
+  ConnectMobileWalletModalText,
+  PayWithEthDescriptionText,
+} from "./PaymentsTextComponents";
+import { PaymentsModal } from "./PaymentsModal";
+import { PaymentsEthForm } from "./PaymentsEthForm";
+
+export enum MODEL_CONTENT {
+  WHY_ETH = "why eth",
+  WHAT_IS_ETH = "what is eth",
+  CAN_USE_CVL = "can I use cvl",
+}
```

Comment:
If you don't care about the values you can use this sugar and just give the key names like here https://github.com/joincivil/Civil/blob/master/packages/components/src/Notice/index.tsx#L6-L12 (I think it just uses the key name as the value?)